src/pylyrics/plot_cloud.py

```diff
@@ -33,6 +34,8 @@ def plot_cloud(
         background color
     show: bool, default=False
         whether to display the plot to screen
+    token: str
```

Comment:
Sorry @artanzand I decided to ditch the Genius token because it is not so stable. Should be just a quick fix by removing the variable.

Reply:
Done!